src/main/webapp/homepage_script.js

```diff
@@ -410,18 +434,18 @@ function fetchTasks(category, cursorAction) {
 function displayTasks(append) {
     if (taskGroup !== null && taskGroup.currentTaskCount > 0) {
         document.getElementById("no-tasks-message").style.display = "none";
-        document.getElementById("loading").style.display = "none";
         document.getElementById("tasks-message").style.display = "block";
-        if (append) document.getElementById("tasks-list").innerHTML += taskGroup.tasks;
+        if (append == true) document.getElementById("tasks-list").innerHTML += taskGroup.tasks;
```

Comment:
Is there a reason for using `append == true` here? I think `if (append)` should perform the same and is better in style.

Reply:
I previously had a different implementation where this wasn't working, but then I changed the implementation and forgot to change this statement. I'll change this in the next commit, thanks!